adder/adderutils/adderutils.go

```diff
@@ -0,0 +1,68 @@
+// Package adderutils provides some utilities for adding content to cluster.
```

Comment:
I not sure this really requires being in its own utils package... Unless I am missing an import cycle, I believe this could simply be a function in the `adder` package.

Reply:
yeah, import cycle :(. adder[utils] -> `local` -> `adder`